source/Ship.cpp

```diff
@@ -4969,3 +4969,36 @@ double Ship::CalculateDeterrence() const
 		}
 	return tempDeterrence;
 }
+
+
+
+bool Ship::Immitates(const Ship &other) const
+{
+	// Some early return checks.
+	if(other.DisplayModelName() != DisplayModelName())
+		return false;
+
+	for(auto oit : outfits)
+		if(other.OutfitCount(oit.first) != OutfitCount(oit.first))
+			return false;
```

Comment:
`outfits` is an `std::map`, so couldn't we just compare the maps?

Reply:
Does that work? You mean using == ?

Reply:
Seems to work